massa-ledger-worker/src/ledger_db.rs

```diff
@@ -585,23 +601,22 @@ impl LedgerDB {
         &self,
         addr: &Address,
     ) -> std::collections::BTreeMap<Vec<u8>, Vec<u8>> {
+        let address_deserializer = AddressDeserializer::new();
         let handle = self.db.cf_handle(LEDGER_CF).expect(CF_ERROR);
 
         let mut opt = ReadOptions::default();
         opt.set_iterate_upper_bound(end_prefix(data_prefix!(addr)).unwrap());
 
         self.db
-            .iterator_cf_opt(
-                handle,
-                opt,
-                IteratorMode::From(data_prefix!(addr), Direction::Forward),
-            )
+            .iterator_cf_opt(handle, opt, IteratorMode::Start)
```

Comment:
Is it possible to re-interoduce a lower-bound, instead of starting from the very beginning, on the db-iterator?